<launch>

  <arg name="cam_param_file"	default=""/>
  <arg name="mk_param_file"	default="$(dirname
					 )/../Data/Logicool640480.txt"/>
  <arg name="marker_frame"	default="marker_frame"/>
  <arg name="image_topic"	default="/camera/image_raw"/>
  <arg name="manager"		default=""/>
  
  <arg if="$(eval manager == '')"	name="command"	value="standalone"/>
  <arg unless="$(eval manager == '')"	name="command"	value="load"/>
  
  <node name="aist_lenti_mark" pkg="nodelet" type="nodelet" output="screen"
  	args="$(arg command) aist_lenti_mark/aist_lenti_mark $(arg manager)">
    <remap from="/image"		to="$(arg image_topic)"/>

    <!-- Other parameters -->
    <param name="cam_param_file"	value="$(arg cam_param_file)"/>
    <param name="mk_param_file"		value="$(arg mk_param_file)"/>
  </node>

</launch>
